AWSAppSyncClient/Internal/AWSMutationRetryAdviceHelper.swift

```diff
@@ -8,14 +8,20 @@ import Foundation
 
 final class AWSMutationRetryAdviceHelper {
     
-    static func isErrorRetriable(error: Error) -> Bool {
+    static func isRetriableNetworkError(error: Error) -> Bool {
```

Comment:
Just to confirm; we will also retry for service level errors, but that retry logic is handled at the network transport layer, right?

I'd like to centralize the retry handling, but I'd guess that's not easily accomplishable, so let's at least document the different retry handling levels some place so it's easier to reason about reviewing the mutation operation code. Perhaps this could be done in the `send` method of the mutation operations, as the top level entry point into the various layers of retry.

Reply:
Yes, you are right. I will update the documentation.